### ISSUE TITLE
[Support]: how to clear regions DB? 

### ISSUE THREAD

**Issue by Mikescotland** [blakeblackshear/frigate#9643]
### Describe the problem you are having

How to clear the regions DB? Changed some parameters in the camera resolution / stream. 

### Version

13rc

### Frigate config file

```yaml
N/a
```

### Relevant log output

```shell
N/a
```

### FFprobe output from your camera

```shell
N/a
```

### Frigate stats

```json
N/a
```

### Operating system

Other Linux

### Install method

Docker Compose

### Coral version

M.2

### Network connection

Wired

### Camera make and model

N/a

**Comment by NickM-27** [comment]
Currently this can only be done manually but in most cases it is not needed because the regions DB holds the data in relative coordinates so when you change resolution the data is still valid.

**Comment by Mikescotland** [comment]
Thanks Nick. I played a bit with contrast and camera fine position so image is not exactly as it was before. I have some problems with one camera, almost 100% of vision is grass, fails to detect a white dog or black clothed person, drives me nuts, hence I played with camera settings and position.
Can I just stop frigate, delete the whole database for frigate and restart docker? I don't really care about history of events etc, as exported required mp4s.
